src/ewatercycle/plugins/hype/forcing.py

```diff
@@ -124,3 +83,41 @@ def to_xarray(self) -> xr.Dataset:
             "history": "Created by ewatercycle.plugins.hype.forcing.HypeForcing.to_xarray()",
         }
         return ds
+
+
+def build_recipe(
```

Comment:
Would it make sense to use model-specific names here? 

```suggestion
def build_hype_recipe(
```

Reply:
suggestion applied.